pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -109,6 +109,7 @@
         # they seem to lead to `too many sections` errors when building base for
         # profiling.
         ++ lib.optionals (!stdenv.targetPlatform.isWindows) [ "split_sections" ]
+        ++ lib.optionals stdenv.targetPlatform.isWindows [ "no_split_sections" ]
```

Comment:
Couldn't build ghc without this.
A bit strange that we were merely _not enabling_ split sections on windows when the comment says they are enabled by default on windows and cause issues

Reply:
Hm we can probably convert this into an `if … else …` thing no w to be more explicit. We don't use the default behavior at any point now.

Reply:
Good point